.npmignore

```diff
@@ -0,0 +1,6 @@
+.circleci/
+lib/
+test/
+index.js
```

Comment:
What's this one for?

Reply:
I meant to ignore the `index.ts`! Nice catch.